website/src/app/pages/data-distributions/data-distributions.component.ts

```diff
@@ -50,7 +170,8 @@ export class DataDistributionsComponent {
   constructor(
     readonly data: DataDistributionsState
   ) {
-    this.spec = this.variable.type === 'Boolean' ? this.createPieSpec(this.variable) : this.createBarSpec(this.variable);
+    this.spec = this.variable.type === 'Boolean' ? this.createPieSpec(this.variable, this.distributionData) : this.createBarSpec(this.variable, this.distributionData);
```

Comment:
Create three demo specs + time slider specs and set them to demoPieSpec, demoBarChartSpec, demoHorizBarChartSpec, timeSpec. Then in the html just create the four specs for demonstration purposes. You should otherwise minimally modify this component as Daniel will be restructuring.